app/src/main/res/values-ru/strings.xml

```diff
@@ -1,49 +1,72 @@
-    <string name="credits_content5">Благодарности  <a href="https://github.com/psyanite">Yan Tsui</a> за тестирование и дизайн иконки.</string>
-    <string name="credits_creator">Создано <a href="https://michael.standen.link">&author;</a></string>
+    <string name="action_credits">О приложении</string>
+
+	<!-- List -->
+    <string name="inaccessible">(недоступен)</string>
+    <string name="go_home_folder">Застряли? Нажмите чтобы вернуться домой</string>
+
+	<!-- Slideshow -->
+    <string name="title_activity_image">Окно изображений</string>
+
     <string name="delete_button">Удалить</string>
-    <string name="delete_dialog_message">Вы уверены?</string>
     <string name="delete_dialog_title">Удалить изображение</string>
-    <string name="go_home_folder">Застряли? Нажмите чтобы вернуться домой</string>
+    <string name="delete_dialog_message">Вы уверены?</string>
     <string name="image_deleted">Изображение удалено</string>
-    <string name="image_description">Изображение</string>
+    <string name="image_not_deleted">Не удалось удалить изображение</string>
+
+    <string name="share_button">Поделиться</string>
+    <string name="share_via">Поделиться с&#8230;</string>
+
     <string name="image_detail_dimensions">Размерности: %1$dx%2$d</string>
-    <string name="image_detail_modified">Модифицирован: %1$s</string>
     <string name="image_detail_size">Размер файла: %1$s</string>
-    <string name="image_not_deleted">Не удалось удалить изображение</string>
-    <string name="inaccessible">Недоступен</string>
+    <string name="image_detail_modified">Модифицирован: %1$s</string>
+
+	<!-- Image descriptions -->
+    <string name="image_description">Изображение</string>
     <string name="logo_description">Логотип</string>
-    <string name="pref_description_image_details">Отобразить информацию изображения во время слайд-шоу.</string>
-    <string name="pref_description_random_order">Отобразить изображения во время слайд-шоу в случайном порядке.</string>
-    <string name="pref_description_reverse_order">Отобразить изображения во время слайд-шоу в обратном порядке.</string>
+
+	<!-- Credits -->
+    <string name="title_activity_credits">О приложении</string>
+    <string name="credits_creator">Создано <a href="https://michael.standen.link">&author;</a></string>
+    <string name="credits_content1">Slideshow это простое приложение для просмотра изображений на Вашем телефоне</string>
+    <string name="credits_content2">Без каких-либо наворотов, это приложение - самое простое приложение для показа слайдов в магазине.</string>
+    <string name="credits_content3">Вы можете посмотреть исходный код приложения, сообщить о проблеме или предложить новые функции на <a href="https://github.com/Screaminghawk/android-slideshow">Github</a>. \t</string>
```

Comment:
Can you please remove the \t at the end of this line? I assume it was added by the translation tool you used, or was it intentional?